src/parsers/timeseries_formats.jl

```diff
@@ -112,7 +112,7 @@ end
 
 This version of the function only has component_name to match the interface. It is unused.
 """
-function read_time_array(
+function read_timeseries(
```

Comment:
Also here, I would suggest renaming to `read_timeseries_file` or something more descriptive.

Reply:
I'm following conventions from many other places like pandas:
pandas.read_csv()
pandas.read_json()